packages/blueprints-integration/src/content.ts

```diff
@@ -52,67 +41,47 @@ export type SomeContent =
 	| CameraContent
 	| RemoteContent
 	| ScriptContent
-	| GraphicsContent
 	| NoraContent
 	| SplitsContent
-	| AudioContent
-	| CameraMovementContent
-	| LowerThirdContent
 	| LiveSpeakContent
-	| MicContent
 	| TransitionContent
 
 export interface VTContent extends BaseContent {
 	fileName: string
 	path: string
-	firstWords: string
-	lastWords: string
-	proxyPath?: string
-	thumbnail?: string
 	loop?: boolean
-	sourceDuration?: number
-	objectDuration?: number
-	metadata?: MetadataElement[]
-	timelineObjects: TimelineObjectCoreExt[]
 	mediaFlowIds?: string[]
 	seek?: number
 	editable?: VTEditableParameters
-	ignoreMediaObjectStatus?: boolean
 }
 
 export interface CameraContent extends BaseContent {
 	studioLabel: string
 	switcherInput: number | string
-	thumbnail?: string
-	timelineObjects: TimelineObjectCoreExt[]
 }
 
 export interface RemoteContent extends BaseContent {
 	studioLabel: string
 	switcherInput: number | string
-	thumbnail?: string
-	timelineObjects: TimelineObjectCoreExt[]
 }
 
 export interface ScriptContent extends BaseContent {
 	firstWords: string
 	lastWords: string
 	fullScript?: string
-	sourceDuration?: number
 	lastModified?: Time | null
 }
 
-export interface GraphicsContent extends BaseContent {
```

Comment:
I think it would be useful to maintain a generic graphics content type for graphics. Going "Nora or Bust" isn't very flexible and may hurt us in the long run. Again, the fact that we don't actually implement that interface in a great way, doesn't mean that we should remove it. Adding rudimentary support for GraphicsContent in hoverscrub would be very simple for anyone wanting to use, for example, Viz integration. I suppose, given what we know, making the `path` property optional might be a good improvement.